tests/test_read_smiles.py

```diff
@@ -615,6 +615,141 @@
          (8, 9, {'order': 1.5})],
         False
     ),
+    # chiral center S/L alanine
+    (
+        'C[C@@H](C(=O)O)N',
+        [(0, {'element': 'C', 'charge': 0, 'aromatic': False, 'hcount': 3}),
+         (1, {'charge': 0, 'aromatic': False, 'element': 'C', 'stereo': (0, 1, 5, 2), 'hcount': 1}),
```

Comment:
this result is incorrect isn't it? since it is implicit the view should be H->C then the neighbors in clockwise fashion. So the resylt should rather be (1, 0, 5, 2) same as below

Reply:
ohh now I get it. I was always interpreting the first atom of the 'accounting' as the one defining the axis. But no you are right in this case. But good that we talked about it.